combat-trainer.lic

```diff
@@ -150,6 +150,8 @@ class LootProcess
     @arrange_types = arrange_types
     @no_skins = []
     @too_many = []
+    @construct = []
```

Comment:
I would recommend renaming this to '@constructs' (note the plural) - this will help you remember that it is an array.